app/views/enrichment_jobs/_enrichment_job.html.erb

```diff
@@ -1,179 +1,181 @@
+      <% if can? :update, @enrichment_job.parser %>
+        <%# <span class="parser-title">: <%= link_to @enrichment_job.parser.try(:name), edit_parser_path(@enrichment_job.parser)></span> %>
+        <span class="parser-title">: <%= link_to @enrichment_job.parser.try(:name), edit_parser_path(@enrichment_job.parser), target: :_top %></span>
+      <% else %>
+        <span class="parser-title">: <%= @enrichment_job.parser.try(:name) %></span>
+      <% end %>
+    </h2>
+
+    <% if @enrichment_job.status == 'active' %>
+      <%=
+        link_to(
+          t("enrichment_jobs.stop"),
           environment_enrichment_job_path(
-            environment: @enrichment_job.environment,
+            environment: params[:environment],
             id: @enrichment_job.id,
+            format: :js,
+            enrichment_job: { status: "stopped" }
           ),
-          params: {
-            enrichment_job: {
-              status: "resume",
-              status_message: "Enrichment was resumed #{l(Time.now, format: :long)}"
-            }
-          },
-          'data-close': '',
           method: :put,
           remote: true,
+          id: "stop-enrichment-button",
           class: "button new-right"
         )
-    } %>
-  <% end %>
-
-  <table class="twelve">
-    <tr>
-      <td><%= t("enrichment_jobs.enrichment") %></td>
-      <td>
-        <strong><%= @enrichment_job.enrichment %></strong>
-      </td>
-    </tr>
-
-    <tr>
-      <td><%= t("enrichment_jobs.record_id") %></td>
-      <td>
-        <strong><%= @enrichment_job.record_id %></strong>
-      </td>
-    </tr>
-
-    <tr>
-      <td><%= t("harvest_jobs.operator") %></td>
-      <td>
-        <% if @enrichment_job.user %>
-          <strong><%= @enrichment_job.user.try(:name) %></strong>
-        <% else %>
-          <strong><%= t("harvest_jobs.scheduled") %></strong>
-        <% end %>
-      </td>
-    </tr>
-
-    <tr>
-      <td><%= t("harvest_jobs.environment") %></td>
-      <td>
-        <strong><%= @enrichment_job.environment %></strong>
-      </td>
-    </tr>
-
-    <tr>
-      <td><%= t("harvest_jobs.start_time") %></td>
-      <td>
-        <% if @enrichment_job.start_time %>
-          <strong><%= l @enrichment_job.start_time, format: :long %></strong>
-        <% end %>
-      </td>
-    </tr>
-
-    <% if @enrichment_job.end_time.present? %>
-    <tr id="enrichment-end-time">
-      <td><%= t("harvest_jobs.end_time") %></td>
-      <td>
-        <strong><%= l @enrichment_job.end_time, format: :long rescue '' %></strong>
-      </td>
-    </tr>
+      %>
+    <% elsif @enrichment_job.status.in?(%w[stopped failed]) %>
+      <button class="button new-right" data-open="resume-confirmation">
+        <%= t('enrichment_jobs.resume') %>
+      </button>
+
+      <%= render partial: 'shared/confirmation_modal', locals: {
+        information: t('enrichment_jobs.resume_modal_information'),
+        id: 'resume-confirmation',
+        yes:
```

Comment:
Can this information be in the template itself? I haven't seen a whole button passed as an argument like this before.

Reply:
Yeah I know it looks a bit strange. Although I quite like the idea to have a shared "confirmation modal" partial instead of using the old and ugly confirmation box from the browser. Do you have suggestions on how to keep the design consistent?